packages/dpns-contract/schema/v1/dpns-contract-documents.json

```diff
@@ -1,5 +1,7 @@
 {
   "domain": {
+    "documentsMutable": false,
```

Comment:
We should do it with data trigger so we can make them updatable easily in future.